tests/system/Entity/EntityTest.php

```diff
@@ -69,11 +69,11 @@ public function testIssetWorksLikeTraditionalIsset()
     {
         $entity = $this->getEntity();
 
-        $this->assertFalse(isset($entity->foo));
+        $this->assertObjectNotHasAttribute('foo', $entity);
 
         $attributes = $this->getPrivateProperty($entity, 'attributes');
-        $this->assertFalse(isset($attributes['foo']));
-        $this->assertTrue(isset($attributes['default']));
+        $this->assertArrayNotHasKey('foo', $attributes);
```

Comment:
This assertion fails. Because `$attributes` is
```
array (4) [
    'foo' => null
    'bar' => null
    'default' => string (6) "sumfin"
    'created_at' => null
]
```
What do we do?

Reply:
You can register `Rector\PHPUnit\Rector\MethodCall\AssertIssetToSpecificMethodRector` into Option::SKIP

```php
use Rector\PHPUnit\Rector\MethodCall\AssertIssetToSpecificMethodRector;

    $parameters->set(Option::SKIP, [
             // ....

            AssertIssetToSpecificMethodRector::class => [
                     __DIR__ . '/tests/system/Entity/EntityTest.php',
           ],
    ]);
```